README.md

```diff
@@ -1,41 +1,39 @@
 
 ## How to publish?
 
 This guide describes the steps of publishing a library built with Kotlin Multiplatform to the [Maven Central repository](https://central.sonatype.com/). To publish your library, you’ll need to:
 
-* Set up credentials, including an account on Maven Central and a PGP key to use for signing
-* Configure the publishing plugin in your library’s project
-* Provide your credentials to the publishing plugin so it can sign and upload your artifacts
-* Run the publication task, either locally or using continuous integration
+* Set up credentials, including an account on Maven Central and a PGP key to use for signing.
+* Configure the publishing plugin in your library’s project.
+* Provide your credentials to the publishing plugin so it can sign and upload your artifacts.
+* Run the publication task, either locally or using continuous integration.
 
 This guide assumes that you are:
 
 - Creating an open-source library.
 - Using macOS or Linux. If you are a Windows user, use [GnuPG or Gpg4win](https://gnupg.org/download) to generate a key pair.
-- Either not registered on Maven Central yet, or have an existing account that’s suitable for [publishing to the Central Portal](https://central.sonatype.org/publish-ea/publish-ea-guide/) (created after March 12th, 2024, or migrated to the Central Portal by their support).
+- Are either not registered on Maven Central yet, or have an existing account that’s suitable for [publishing to the Central Portal](https://central.sonatype.org/publish-ea/publish-ea-guide/) (created after March 12th, 2024, or migrated to the Central Portal by their support).
 - Publishing your library in a GitHub repository.
 - Using GitHub Actions for continuous integration.
 
 Most of the steps here are still applicable if you’re using a different setup, but there might be some differences you need to account for. An [important limitation](https://kotlinlang.org/docs/multiplatform-publish-lib.html#host-requirements) is that Apple targets must be built on a machine with macOS.
 
-Throughout this guide, we’ll use the [https://github.com/kotlin-hands-on/fibonacci](https://github.com/kotlin-hands-on/fibonacci) repository as an example. You can refer to the code of this repository to see how the publishing setup works. Don’t forget to **replace all example values with your own** as you’re configuring your project.
+Throughout this guide, we’ll use the [https://github.com/kotlin-hands-on/fibonacci](https://github.com/kotlin-hands-on/fibonacci) repository as an example. You can refer to the code of this repository to see how the publishing setup works. You **must replace all example values with your own** as you’re configuring your project.
 
 ### Prepare accounts and credentials
 
-#### Register a namespace {#register-a-namespace}
+#### Register a namespace
```

Comment:
There is a link to this section lower down in the doc - is this not needed?

Reply:
Nope :)

Maybe it's necessary in a different Markdown syntax, but for GitHub it's not needed, and [it looks strange](https://github.com/Kotlin/multiplatform-library-template?tab=readme-ov-file#register-a-namespace-register-a-namespace):

![Uploading image.png…]()